scripts/ci/java-test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+cd ../.. || exit 1
+
+failures=""
+
+docker build -t criu-openj9-ubuntu-test:latest -f scripts/build/Dockerfile.openj9-ubuntu .
+if ! docker run --rm --privileged criu-openj9-ubuntu-test:latest; then
+	failures="$failures ubuntu"
```

Comment:
It might be good to use `openj9-ubuntu` and `hotspot-ubuntu` to be able to distinguish between both errors.

Reply:
Agree, fixed.